caddyhttp/httpserver/plugin.go

```diff
@@ -447,6 +447,8 @@ var directives = []string{
 	"realip", // github.com/captncraig/caddy-realip
 	"git",    // github.com/abiosoft/caddy-git
 
+	"cache",
```

Comment:
If being lower in the list will not serve its purpose well, then it's okay to keep it higher. I can definitely understand wanting to keep it above gzip so you can cache compressed responses where clients accept the same encoding.

As long as URL rewrites don't ever have a 1:many mapping, putting cache before rewrite should be fine.

Just don't put it in its own space. ;) The whitespace (empty lines) seem to signal that cache is on its own and I think it's best if it fits in as close as it can with existing groups.